src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -91,6 +92,13 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchCurrentUser(currentUser);
     }
 
+    componentDidUpdate({ refresh: prevRefresh }: Props) {
+        const { refresh } = this.props;
+        if (refresh !== prevRefresh) {
```

Comment:
This will get recalled if you toggle back to false - how can this work more than once? It sounds like you want a ref, `key`, or move state up a level. Or perhaps a random value?

- https://reactjs.org/docs/refs-and-the-dom.html#adding-a-ref-to-a-class-component
- https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recommendation-fully-uncontrolled-component-with-a-key

Reply:
As we discussed offline, the value itself is not important, just a trigger to detect when refresh should occur. Will change the `setState` in `ContentSidebar` to use the updater function instead so async setState's batched together won't fail to trigger an update